Gui/opensim/view/src/org/opensim/view/pub/ViewDB.java

```diff
@@ -1916,16 +1910,12 @@ public static void ApplyFunctionToActors(vtkProp3D asm, ActorFunctionApplier fun
    }
 
    public double getNonCurrentModelOpacity() {
-         String nonCurrentModelOpacityStr = NbBundle.getMessage(ViewDB.class,"CTL_NonCurrentModelOpacity");
-         nonCurrentModelOpacityStr=Preferences.userNodeForPackage(TheApp.class).get("NonCurrentModelOpacity", nonCurrentModelOpacityStr);
-         if (nonCurrentModelOpacityStr != null)
-            setNonCurrentModelOpacity(Double.valueOf(nonCurrentModelOpacityStr));
          return nonCurrentModelOpacity;
    }
 
    public void setNonCurrentModelOpacity(double nonCurrentModelOpacity) {
       this.nonCurrentModelOpacity = nonCurrentModelOpacity;
-      Preferences.userNodeForPackage(TheApp.class).get("NonCurrentModelOpacity", String.valueOf(nonCurrentModelOpacity));
+      TheApp.getCurrentVersionPreferences().get("NonCurrentModelOpacity", String.valueOf(nonCurrentModelOpacity));
```

Comment:
"Internal.NonCurrentModelOpacity"?

Reply:
This one is obsolete (vtk) and will be removed when Branch (https://github.com/opensim-org/opensim-gui/tree/remove_dead_code) is merged, for now I'll mark it Internal to avoid clutter. Thank you.